src/api/common/schemas/pathfind.json

```diff
@@ -24,7 +24,32 @@
       "additionalProperties": false,
       "required": ["address"]
     },
-    "destination": {"$ref": "adjustment"}
+    "destination": {
+      "type": "object",
+      "properties": {
+        "address": {"$ref": "address"},
+        "amount": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "description": "The quantity of the currency, denoted as a string to retain floating point precision",
+              "$ref": "value"
+            },
+            "currency": {
+              "description": "The three-character code or hex string used to denote currencies",
+              "$ref": "currency"
+            },
+            "counterparty": {
+              "description": "The Ripple account address of the currency's issuer or gateway",
+              "$ref": "address"
+            },
+            "required": ["currency", "value"]
```

Comment:
So this change is to make counterparty optional? Is there documentation to confirm that the destination counterparty is optional for non-XRP?